src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -260,8 +261,18 @@ internal static string ParameterizedName(IEdmOperation operation)
             return sb.ToString();
         }
 
+        internal static bool TryGetNamespaceNameFromQualifiedName(string qualifiedName, out string namespaceName, out string name, out string fullName)
+        {
+            bool foundNamespace = EdmUtil.TryGetNamespaceNameFromQualifiedName(qualifiedName, out namespaceName, out name);
+
+            fullName = EdmUtil.GetFullNameForSchemaElement(namespaceName, name);
+            return foundNamespace;
+        }
+
         internal static bool TryGetNamespaceNameFromQualifiedName(string qualifiedName, out string namespaceName, out string name)
         {

```

Comment:
We should be able to revert changes to this version of the overload, yes?  i.e., rather than introduce the foundNamespace, boolean, we can just return true or false in each case below.